command/debug/debug.go

```diff
@@ -373,43 +373,32 @@ func (c *cmd) captureInterval(ctx context.Context) error {
 
 func captureShortLived(c *cmd) error {
 	g := new(errgroup.Group)
-	timestamp := time.Now().Local().Unix()
 
-	timestampDir, err := c.createTimestampDir(timestamp)
+	dir, err := makeTimestampDir(c.output, c.timeNow())
 	if err != nil {
 		return err
 	}
 	if c.captureTarget(targetProfiles) {
 		g.Go(func() error {
-			return c.captureHeap(timestampDir)
+			return c.captureHeap(dir)
 		})
 
 		g.Go(func() error {
-			return c.captureGoRoutines(timestampDir)
+			return c.captureGoRoutines(dir)
 		})
 	}
 	return g.Wait()
 }
 
-func (c *cmd) createTimestampDir(timestamp int64) (string, error) {
-	// Make the directory that will store all captured data
-	// for this interval
-	timestampDir := fmt.Sprintf("%s/%d", c.output, timestamp)
-	err := os.MkdirAll(timestampDir, 0755)
-	if err != nil {
-		return "", err
+func makeTimestampDir(base string, now time.Time) (string, error) {
```

Comment:
We should continue to get rid of `Timestamp`, what about `makeCaptureDir`?

Reply:
yes that would work too. I also saw some mentions of timestamp as var names

Reply:
Good catch, fixed those now as well